package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "spdx-compare",
+  "name": "@metamask/spdx-compare",
   "description": "compare SPDX license expressions",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Seems like all of the changes are fixes. Should this be 1.0.1?

Alternatively, conventionally we've bumped major when renaming. Should this be 2.0.0?

Reply:
Yeah, let's go with 2.0.0 just so it's one more thing that stands out with this release.